src/main/java/io/reactivex/internal/operators/flowable/BlockingFlowableIterable.java

```diff
@@ -125,9 +125,7 @@ public T next() {
 
         @Override
         public void onSubscribe(Subscription s) {
-            if (SubscriptionHelper.setOnce(this, s)) {
-                s.request(batchSize);
-            }
+            SubscriptionHelper.setOnce(this, s, batchSize);
```

Comment:
When the `request` call comes directly after `setOnce`, they can be combined into one and this resolves a lot of partial coverage cases.